src/app/pages/forms/components/inputs/components/datetimepicker/datetimepicker.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, ElementRef, Input, Output, EventEmitter, OnInit, NgZone, SimpleChange} from '@angular/core';
+
+
+@Component({
+  selector: 'datetimepicker',
+  template: require('./datetimepicker.html')
+})
+
+export class DateTimePicker extends OnInit{
+  @Input() value: string;
+  @Input() startDate: any;
+  @Input() endDate: any;
+  @Input() minuteStep = 5;
+  @Input() format: string = 'dd-mm-yyyy hh:ii';
+  @Input() disableType: boolean = false;
+  @Output() valueChange: EventEmitter<string> = new EventEmitter<string>();
+
+  private _picker = null;
+
+  constructor(private myElement: ElementRef, private zone: NgZone) {
+    super();
+  }
+
+  ngOnInit():any {
+    this._picker = jQuery(this.myElement.nativeElement).find('.datetime-picker');
```

Comment:
I guess there is no need in jQuery selector here as nativeElement should already be a proper object for $().datetimepicker call.